app/shared/connectors/httpparsers/HttpParser.scala

```diff
@@ -53,22 +53,27 @@ trait HttpParser extends Logging {
 
   private val multipleErrorReads: Reads[Seq[DownstreamErrorCode]] = (__ \ "failures").read[Seq[DownstreamErrorCode]]
 
+  private val multipleHipErrorReads: Reads[Seq[HipDownstreamErrorCode]] = (__ \ "failures").read[Seq[HipDownstreamErrorCode]]
```

Comment:
needs to read the response object first ie. 
`(__ \ "response" \ "failures").read[Seq[HipDownstreamErrorCode]]`